doc/lib.rst

```diff
@@ -18,9 +18,9 @@ low-level interface to a C library.
 
 Read this `document <apis.html>`_ for a quick overview of the API design.
 
-The `bottom <#nimble>`_ of this page includes a list of 3rd party packages
-created by the Nim community. These packages are a useful addition to the
-modules in the standard library.
+In addition to the modules in the standard library, third-party packages
+created by the Nim community can be used via `Nimble <#nimble>`_, Nim's
```

Comment:
This should use full URL.

Reply:
Full URL to what? It's just linking to the anchor at the bottom of the page and the way it's written now allows it to work on the live site, devel docs, & the `*.rst` file viewed in GitHub.

Or maybe we should remove the Nimble section from the bottom of the page entirely and move its content here.

Reply:
Oh ok, I see what you mean.